src/components/BigClickableButton/BigClickableButton.test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import React from 'react';
+import { mount } from 'enzyme';
+import BigClickableButton from './BigClickableButton';
+// Use Lolex to mock setTimeout - later use Jest mock once PR https://github.com/facebook/jest/pull/5171 landed
+import lolex from 'lolex';
+
+describe('BigClickableButton component', () => {
+  let wrapper;
+  let button;
+  lolex.install(); // mock setTimeout
```

Comment:
Do we need to `uninstall` after this is done? Not familiar with lolex but they are uninstalling in each example.

Reply:
I'm also not familiar with Lolex but I think it's OK here to not call `uninstall` as we're only needing a static value for `Date.now()`.

But if we would use it to test a spring at different tick positions we should add install to `beforeEach` and `uninstall` in `afterEach` - so we're having a fresh timer mock at each test. 

I haven't tested a spring animation yet but `clock.next()` seems the way to test the next fired timeout event where `clock = lolex.install()` or `clock.tick(100)` to increment the timer by 100ms.